### ISSUE TITLE
My PI node is not working. I have installed docker and whenever I start PI node its's showing attached screen. Which is not going ahead from this.

### ISSUE THREAD

**Issue by tapan15** [pi-node/instructions#220]
![pi](https://user-images.githubusercontent.com/85295346/147759109-974b9d38-0128-44e3-aede-ba8e2a2b7727.jpg)

**Comment by sepehrrostami** [comment]
Hi.👋
The reason for this is still unknown and the moderators have not said anything about it. This happened to me a few months ago and it was fixed a few days later.

**Comment by jspxhdyp** [comment]
你可以下载使用中国版1.4.7版本，它是正常的。

**Comment by jspxhdyp** [comment]
你可以下载使用0.4.7版本，他是正常的。

**Comment by tapan15** [comment]
Thank you ! can you provide the download link for the version 0.4.7 ? On the website there is only 0.4.5 version available.
Thanks

**Comment by dmciv010** [comment]
Glad to see I'm not the only one... If we encounter this problem (which I have), should we leave the app open and running as shown in the screenshot, or just shut it down and try again every few days? Thank you!

**Comment by ayvc0420** [comment]
me too.

**Comment by sepehrrostami** [comment]
I closed the app and after some days it was fixed.
before closing it, I tried to open the app with some kind of VPN

**Comment by AndyKr0p** [comment]
> Thank you ! can you provide the download link for the version 0.4.7 ? On the website there is only 0.4.5 version available. Thanks

Do you really want to enter your credentials in Pi Network app which is not from the official website?! Maaan, you need to pass a basic course about cyber security to cry at least less in the future.

**Comment by tapan15** [comment]
Nope! Am just checking is there scamers around?!! BTW am waiting for the solution if someone find pls update it here. May be mainnet is live so software/app is in upgrading mode? Let's see if in a week we get the solution from the dev team.

**Comment by ayvc0420** [comment]
0.4.7 is the official version of the file, unfortunately it is not provided on the official website, it is provided for the Chinese region.

**Comment by ayvc0420** [comment]
![Screenshot_20220103-031438_Chrome](https://user-images.githubusercontent.com/88650033/147886889-b472fe83-1685-4b91-99e6-a68d4c08db7d.jpg)

**Comment by ihatejam** [comment]
Anyone in China will need to run version 0.4.7 which was made available specifically to work with the firewall challenges a few months ago (this version should not be used by anyone who is not behind the Great Firewall). 
Please ask a moderator for the download link in the Chinese chat room inside the Pi app (Please take the link from moderators only).

**Comment by ihatejam** [comment]
**Update**: It turns out that all new nodes installed since December 18 have this issue. The Core Team is currently focussing on completing their Mainnet initialization tasks and will then refocus on fixing this node issue afterward.

**Comment by sepehrrostami** [comment]
> 

you know, it is not an issue. They put all new nodes in this mode for a reason. reason:still unknown
